wemulate/main.py

```diff
@@ -1,4 +1,9 @@
 from typing import Optional
+
```

Comment:
blank line

Reply:
For me it is more readable because the external dependencies are separated to the internal imports. Thoughts?